src/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -93,14 +93,22 @@ public async Task<bool> ProcessPayload(WorkflowRequestEvent message)
 
             var workflowInstances = new List<WorkflowInstance>();
 
-            workflows.ForEach(async (workflow) => workflowInstances.Add(await CreateWorkflowInstanceAsync(message, workflow)));
+            workflows.ForEach(async (workflow) =>
+            {
+                var workflowInstance = await CreateWorkflowInstanceAsync(message, workflow);
```

Comment:
any particular reason to split this out?

Reply:
Just felt like it made the code a little clearer the CreateWorkflowInstance was all way on right when its an important part of the code.